### ISSUE TITLE
why mask not worked?

### ISSUE THREAD

**Issue by RobinChen007** [TencentARC/BrushNet#23]
i have used the mask to generate background, but the model change my foreground !
![13b69ddd0d08c](https://github.com/TencentARC/BrushNet/assets/53717420/741ee582-8126-4002-b4e1-f0bbe626a703)
![dfbada49e0035](https://github.com/TencentARC/BrushNet/assets/53717420/a07b0953-b954-4655-91d3-f542ad07611b)

the result is 
![output20240426061019](https://github.com/TencentARC/BrushNet/assets/53717420/ee6274ae-6798-4898-93a7-4727006840ec)
you can find the words in foreground is changed!

**Comment by juxuan27** [comment]
Hi, you need to open the blend option to ensure exact same foreground.

**Comment by RobinChen007** [comment]
> Hi, you need to open the blend option to ensure exact same foreground.

yes, as you said, the mask layer is in effect, but I find another question ... ...
you can find the edge of the image, the images generated by models Does not match the original image，
![Uploading res.png…]()

**Comment by juxuan27** [comment]
You may change the hyperparameter of the blend option to see if it helps. [here](https://github.com/TencentARC/BrushNet/blob/ef14cadf3608560376f827e9e978b204ca44fd6f/examples/brushnet/test_brushnet.py#L61)
